api/seeds/shared/set-up-users.js

```diff
@@ -128,6 +130,7 @@ const createUsersToAdd = async (knex, oktaClient) => {
       state_id: 'md',
       role_id: stateStaffRoleId,
       status: 'approved',
+      expires_at: format(new Date(new Date().getFullYear() + 1, '06', '30'), PostgresDateFormat),
```

Comment:
This is a state staff, is expiring june 30 of next year right?  It might be, but this is a split role situation and they are not an admin for MD.

Reply:
Whoops! good catch